extensions/wikipedia/src/featured-page.tsx

```diff
@@ -3,6 +3,6 @@ import { closeMainWindow, open } from "@raycast/api";
 
 export default async function () {
   const pageTitle = await getTodayFeaturedPageTitle();
-  await open(`https://wikipedia.org/wiki/${pageTitle}`);
+  await open(`https://wikipedia.org/wiki/${pageTitle.replaceAll(" ", "_")}`);
```

Comment:
according to https://stackoverflow.com/a/63492081, we might have to `encodeURIComponent` it as well

Reply:
done